.eslintrc.cjs

```diff
@@ -17,6 +17,7 @@ module.exports = {
             'XMLHttpRequest'
         ]
     },
+    parser: '@babel/eslint-parser',
```

Comment:
What language feature necessitated this?

Reply:
eslint gives this otherwise

```
JSONPath/rollup.config.js
  2:27  error  Parse errors in imported module '@rollup/plugin-node-resolve': Unexpected token { (25:11)  import/no-deprecated
  2:27  error  Parse errors in imported module '@rollup/plugin-node-resolve': Unexpected token { (25:11)  import/namespace

✖ 2 problems (2 errors, 0 warnings)
```

Not sure if there's some other better way to fix it or not.

Reply:
Instead of `parser`, try adding `parserOptions: {ecmaVersion: 2022}` and see if that resolves, or failing that, `parserOptions: {ecmaVersion: 2022, sourceType: 'module'}`.

Reply:
That worked.
Thanks !